src/main/_docs/setup/setup-getting-started.md

```diff
@@ -208,7 +208,7 @@ We will attempt to auto-set `CHE_HOST` by running an internal utility `docker ru
 docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Refer to cloud setup at top of section with link to {{base}}{{site.links["setup-bitnami"]}} .